experimental/BasisLieHighestWeight/src/MainAlgorithm.jl

```diff
@@ -308,7 +308,19 @@ function add_new_monomials!(
       zero_coordinates,
     ),
   )
-  isempty(poss_mon_in_weightspace) && error("The input seems to be invalid.")
+  if isempty(poss_mon_in_weightspace)
+    if V isa SimpleModuleData
+      error(
+        "The input seems to be invalid. Not enough monomials for weightspace $(highest_weight(V) - weight_w) in module with highest weight $(highest_weight(V))"
+      )
+    elseif V isa DemazureModuleData
+      error(
+        "The input seems to be invalid. Not enough monomials for weightspace $((highest_weight(V) - weight_w) * weyl_group_elem(V)) in module with extremal weight $(highest_weight(V) * weyl_group_elem(V))"
```

Comment:
```suggestion
        "The input seems to be invalid. Not enough monomials for weightspace $((highest_weight(V) - weight_w)) * $(weyl_group_elem(V)) in module with extremal weight $(highest_weight(V)) * $(weyl_group_elem(V))"
```
in most other places you print these as a formal product, and don't actually compute the product. is there a reason you do it differently here?

Reply:
It was the most helpful for debugging